test/e2e/tests/settings.spec.js

```diff
@@ -0,0 +1,109 @@
+const { strict: assert } = require('assert');
+const { convertToHexValue, withFixtures } = require('../helpers');
+
+describe('Settings search feature', function () {
+  const ganacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: convertToHexValue(25000000000000000000),
+      },
+    ],
+  };
+  it('should successfully find advanced settings option', async function () {
+    await withFixtures(
+      {
+        fixtures: 'imported-account',
+        ganacheOptions,
+        title: this.test.title,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+
+        // goes to settings page
+        await driver.clickElement('.account-menu__icon');
+        const settingsButton = await driver.findClickableElement({
+          tag: 'div',
+          text: 'Settings',
+        });
+        assert.equal(await settingsButton.getText(), 'Settings');
```

Comment:
I don't think this assertion is necessary, since we are finding the element using the text `Settings`